File: Tema2/Ejercicio03/Ejercicio3.java
import java.util.Scanner;

public class Ejercicio3 {
    
    public static void main(String[] args) {
        Scanner lector = new Scanner(System.in);
        System.out.println("Escribe un numero cercano al cero: ");
        double numeroInput = lector.nextDouble();
        boolean funcionando = true;
        while (funcionando == true){
            if (numeroInput > 0 && numeroInput < 1){
                System.out.println("Este numero es Casi 0");
                funcionando = false;
            }else if (numeroInput < 0 && numeroInput > -1) {
                System.out.println("Este numero es Casi 0");
                funcionando = false;
            }else {
                System.out.println("Sigue intentando con otro numero más cercano a 0");
            }
        }
    lector.close();
        
    }
}

File: Tema2/Ejercicio03/Ejercicio3.py
#coding: latin1

from decimal import Decimal
from tokenize import Double


class Ejercicio3:
    numeroInput = Decimal(input("Introduce un numero para comprobar como es: "))
    seguir = True
    while (seguir == True):
        if ((numeroInput > 0) and (numeroInput < 1)):
            print ("El numero es Casi 0")
            seguir = False
        elif ((numeroInput < 0) and (numeroInput > -1)):
            print ("El numero es Casi 0") 
            seguir = False
        else:
            numeroInput = Decimal(input("Sigue intentandolo con otro numero que sea casi 0: "))
        numeroInput